src/zoid/buttons/component.jsx

```diff
@@ -716,9 +717,16 @@ export const getButtonsComponent: () => ButtonsComponent = memoize(() => {
             messageType,
             offerCountryCode,
             creditProductIdentifier,
+            clickUrlOverride,
           }) => {
-            const { message, clientID, merchantID, currency, buttonSessionID } =
-              props;
+            const {
+              message,
+              clientID,
+              merchantID,
+              currency,
+              buttonSessionID,
+              merchantRequestedPopupsDisabled,
```

Comment:
This implementation was inspired by an [existing popup implementation](https://github.com/paypal/paypal-checkout-components/blob/04cca79a5ebcd7893488bc77fca214897452de78/src/zoid/buttons/prerender.jsx#L64) and thought it would be best to add its' considerations to this feature as well.  Glad you found utility for it elsewhere!